src/track/beats.h

```diff
@@ -41,7 +41,8 @@ class Beats : public QObject {
         BEATSCAP_TRANSLATE     = 0x0002, // Move all beat markers earlier or later
         BEATSCAP_SCALE         = 0x0004, // Scale beat distance by a fixed ratio
         BEATSCAP_MOVEBEAT      = 0x0008, // Move a single Beat
-        BEATSCAP_SETBPM        = 0x0010  // Set new bpm, beat grid only
+        BEATSCAP_SETBPM        = 0x0010, // Set new bpm, beat grid only
```

Comment:
Distinguishing between setting the BPM manually or setting it to an implicit value by rounding doesn't matter. I suggest renaming `BEATSCAP_SETBPM` to `BEATSCAP_ADJUSTBPM` covering both `BEATSCAP_SETBPM` and `BEATSCAP_ROUND`.

Reply:
The reason I implemented it this way is, that the upcoming beats class could implement a rounding based on segments as written in the later comment. Rounding in a non constant beatgrid class would mean finding segments that seems constant and round those to integer BPM.